packages/ui/src/components/eventTimelineWidget.js

```diff
@@ -30,30 +31,41 @@ class EventTimelineWidget extends Component {
     }
 
     /**
+     * @name loadEventDetails
+     * @desc Move to event details page
+     * @return Loads the event details page
+     */
+    loadEventDetails = (id) => {
+      console.log(id)
+      History.push(`/eventDetails/${id}`)
```

Comment:
This is wrong. You are not supposed to route it this way. these routes should be done by Link

Reply:
Made changes as per you comment